pkg/virt-launcher/virtwrap/network/podnic.go

```diff
@@ -195,11 +200,20 @@ func (l *podNIC) PlugPhase1() error {
 		log.Log.V(4).Infof("The generated dhcpConfig: %s", dhcpConfig.String())
 		if err := l.cacheFactory.CacheDHCPConfigForPid(getPIDString(l.launcherPID)).Write(l.podInterfaceName, dhcpConfig); err != nil {
 			log.Log.Reason(err).Error("failed to save dhcpConfig configuration")
-			return errors.CreateCriticalNetworkError(err)
```

Comment:
I don't understand why you remove this error - if we fail to generate the dhcp configuration for bridge binding (the only one w/ a non-recoverable config) should we crash & burn ? Not sure if we would be in a recoverable state actually.

Reply:
Returning nothing is wrong, but I think we should return here a non critical error so the generation will be retried (we haven't prepare the networking so we can retry it).

Reply:
Fair enough.